sdk/src/ingredient.rs

```diff
@@ -895,18 +899,11 @@ impl Ingredient {
                 Err(jumbf_store.unwrap_err())
             };
 
-            (
-                result.inspect_err(|e| {
-                    // add a log entry for the error so we act like verify
-                    log_item!("asset", "error loading file", "Ingredient::from_file")
-                        .failure_no_throw(&mut validation_log, e);
-                }),
-                Some(manifest_bytes),
-            )
+            (result, Some(manifest_bytes))
         } else {
             // This will always be Err in this situation
```

Comment:
You could rewrite the `if` at line 879 as a `match` statement and that would eliminate the need for the `unwrap` here.

Reply:
Thanks, I this is much cleaner now.